mms/tests/integration_tests/onnx_export_integ_test.py

```diff
@@ -34,46 +59,87 @@ def _download_file(download_dir, url):
 
 def cleanup(tmpdir):
     print("Deleting all downloaded resources for SSD MXNet Model Server Integration Test")
-    shutil.rmtree(tmpdir)
+    shutil.rmtree(str(tmpdir))
 
 
-def setup_onnx_integ(tmpdir):
+def setup_onnx_integ(tmpdir, URL,port):
     # Start the mxnet model server for onnx integ tests.
     print("Starting MXNet Model Server for onnx integ test..")
 
     # Set argv parameters
     sys.argv = ['mxnet-model-server']
-    sys.argv.append("--models")
-    sys.argv.append("squeezenet={}/squeezenet.model".format(tmpdir))
+    sys.argv.append('--models')
+    for onnx_model in URL.keys():
+        sys.argv.append("{}={}/{}.model".format(onnx_model, tmpdir, onnx_model))
+    sys.argv.append('--port')
+    sys.argv.append(port)
     mxnet_model_server.start_serving()
 
 
-def create_model(tmpdir):
+def create_model(tmpdir,URL, onnx_source_model_zoo = True):
     # Download the files required for onnx integ tests.
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/squeezenet.onnx")
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/synset.txt")
-    # Download input image.
-    _download_file(tmpdir, "https://upload.wikimedia.org/wikipedia/commons/8/8f/Cute-kittens-12929201-1600-1200.jpg")
-    # Export the model.
-    print("Exporting the mxnet model server model...")
-    subprocess.check_call(['mxnet-model-export', '--model-name', 'squeezenet', '--model-path', tmpdir], cwd=tmpdir)
+    download_dir = tmpdir + '/scratch'
+    try:
+        os.mkdir(download_dir)
+        for onnx_model in URL.keys():
+            if onnx_source_model_zoo:
+                _download_file(download_dir, URL[onnx_model])
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/synset.txt")
+                model_tar = '{}/{}.tar.gz'.format(download_dir,onnx_model) 
+                tar = tarfile.open(model_tar, "r:*")
+                tar.extractall(path=download_dir)
+                tar.close()
+                model_dir = '{}/{}'.format(download_dir, onnx_model)
+                model_path = os.path.join(model_dir, 'model.onnx')
+                new_path = os.path.join(model_dir, '{}.onnx'.format(onnx_model))
+                os.rename(model_path, new_path)
+                shutil.move(new_path, download_dir)
+                print('model dir is {}'.format(os.listdir(model_dir)))
```

Comment:
Are these prints for debugging or are they a part of the commit?

Reply:
@vdantu Though this prints are currently only for debug, I need to add more prints as mentioned by Sandeep in previous comments